src/python/gudhi/cubical_complex.pyx

```diff
@@ -139,6 +148,33 @@ cdef class CubicalComplex:
         """
         return self.thisptr.dimension()
 
+    def all_cells(self):
+        """Array with the filtration values of all the cells of the complex.
+        Modifying the values is strongly discouraged.
+
+        :returns:  numpy.ndarray
+        """
+        cdef np.npy_intp dim = self.thisptr.data.size()
+        a = np.PyArray_SimpleNewFromData(1, &dim, np.NPY_DOUBLE, self.thisptr.data.data())
+        np.set_array_base(a, self)
+        return a.reshape([2*d+1 for d in self.thisptr.shape()], order='F')
+
+    def top_cells(self):
```

Comment:
Maybe this should be called `top_dimensional_cells` for consistency with the constructor argument, although I find that name too long.